tests/jballerina-unit-test/src/test/resources/test-src/statements/vardeclr/module_tuple_var_decl.bal

```diff
@@ -79,6 +79,25 @@ function testDeclaredWithVar2() {
     assertEquality(13, restBp[1]);
 }
 
+var [[intVal], {a: intVal2}, ...otherValues] = getComplexTuple();
+
+function testDeclaredWithVar3() {
+    assertEquality(5, intVal);
+    assertEquality(6, intVal2);
+    int|error err = otherValues[0];
+    if (err is error) {
+        assertEquality("error msg", err.message());
+    }
```

Comment:
We need to handle the else scenario and fail here, right? Please check wherever applicable.